ConsistencyManager/ConsistencyManager.swift

```diff
@@ -74,14 +74,14 @@ public class ConsistencyManager {
      It's useful if you want to add timers to this work.
      Called on an internal thread.
     */
-    public static let kCleanMemoryAsynchronousWorkStarted = "com.linkedin.consistencyManager.kCleanMemoryAsynchronousWorkStarted"
+    open static let kCleanMemoryAsynchronousWorkStarted = Notification.Name("com.linkedin.consistencyManager.kCleanMemoryAsynchronousWorkStarted")
```

Comment:
probably something for a future update, but you can't override something that is `static`so, technically, this should probably be `public`

same for other stuff that is `static`

it's kind of weird that this doesn't trigger warnings, actually